static/js/keydown_util.js

```diff
@@ -25,12 +25,15 @@ export function handle(opts) {
             return;
         }
 
-        const key_name = keys[key];
+        function valid_key(key) {
+            return key in keys;
+        }
 
-        if (!key_name) {
+        if (!valid_key(key)) {
```

Comment:
Do we need a function, or could we just do `if (!(key in keys))`?  I guess it's kinda awkward in JS because of how `!` scoping works.

Reply:
AFAIK yes, we need a function here. I tried that and `Object.hasOwnProperty` checks and it doesn't work. Normally, we wouldn't need a function for general types like a `string` or a `number`.